File: vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/main/java/com/vaadin/flow/component/combobox/test/template/ComboBoxInATemplate2.java

package com.vaadin.flow.component.combobox.test.template;

import com.vaadin.flow.component.Tag;
import com.vaadin.flow.component.combobox.ComboBox;
import com.vaadin.flow.component.dependency.JsModule;
import com.vaadin.flow.component.polymertemplate.Id;
import com.vaadin.flow.component.polymertemplate.PolymerTemplate;
import com.vaadin.flow.templatemodel.TemplateModel;

@Tag("combo-box-in-a-template2")
@JsModule("./src/combo-box-in-a-template2.js")
public class ComboBoxInATemplate2 extends PolymerTemplate<TemplateModel> {

    @Id("comboBox2")
    ComboBox<String> comboBox;

    public ComboBox<String> getComboBox() {
        return comboBox;
    }
}

File: vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/frontend/src/combo-box-in-a-template2.js
import {PolymerElement} from '@polymer/polymer/polymer-element.js';
import {html} from '@polymer/polymer/lib/utils/html-tag.js';

class ComboBoxInATemplate2 extends PolymerElement {
  static get template() {
    return html`
        <vaadin-combo-box id="comboBox2"></vaadin-combo-box>
`;
  }

  static get is() {
      return 'combo-box-in-a-template2'
  }
}
customElements.define(ComboBoxInATemplate2.is, ComboBoxInATemplate2);
